src/main/java/core/mate/academy/service/impl/ExcavatorProducerImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.mate.academy.service.impl;
+
+import core.mate.academy.model.Excavator;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducerImpl implements MachineProducer<Excavator> {
```

Comment:
The class implementation should not start with an empty line. Please remove the empty line at the beginning of the class definition.